src/openzaak/tests/management/test_zaakeigenschap_command.py

```diff
@@ -28,7 +28,9 @@ def test_valid(self):
             zaak__zaaktype=eigenschap.zaaktype,
         )
 
-        call_command("check_zaak_eigenschappen", stdout=stdout, stderr=stderr)
+        call_command(
+            "check_zaak_eigenschappen", stdout=stdout, stderr=stderr, no_color=True
```

Comment:
Why `no_color`? What does it do?

Reply:
Some commands can use specific styling for command output (green for succes, red for failure: https://docs.djangoproject.com/en/5.1/howto/custom-management-commands/#django.core.management.BaseCommand.style), which makes it annoying to do assertions on the output. This was failing for me locally for some reason so that's why I forced `no_color=True`